build.sbt

```diff
@@ -306,7 +306,9 @@ val mimaFilterSettings = Seq {
     ProblemFilters.exclude[MissingClassProblem]("scala.collection.immutable.VectorMap$Tombstone$NextOfKin$"),
     ProblemFilters.exclude[MissingClassProblem]("scala.collection.immutable.VectorMap$Tombstone$Kinless$"),
     ProblemFilters.exclude[MissingTypesProblem]("scala.collection.immutable.VectorMap$Tombstone$"),
-    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.immutable.VectorMap#Tombstone.unapply")
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.immutable.VectorMap#Tombstone.unapply"),
+
+    ProblemFilters.exclude[DirectMissingMethodProblem]("scala.collection.immutable.HashMapBuilder.result"),
```

Comment:
Note to self: a safe forward-compatibility breakage, because `HashMapBuilder` is package-private (previously to `immutable` now to `collection`) (for completeness, package-private isn't a guarantee of universal safety, with type/val aliasing, but it's assumed this one isn't being leaked).